packages/server-core/src/social/invite/invite.hooks.ts

```diff
@@ -66,14 +155,16 @@ export default {
       () => schemaHooks.validateData(invitePatchValidator),
       schemaHooks.resolveData(invitePatchResolver)
     ],
-    remove: [authenticate(), iff(isProvider('external'), inviteRemoveAuthenticate())]
+    remove: [authenticate(), iff(isProvider('external'), inviteRemoveAuthenticate()), removeFriend]
   },
 
   after: {
     all: [],
     find: [],
     get: [],
-    create: [],
+    create: [
+      (context: HookContext<InviteService>) => sendInvite(context.app, context.result as InviteType, context.params)
```

Comment:
The `sendInvite` method can be updated to be a hook instead of method which it currently is.